lib/internal/repl.js

```diff
@@ -59,7 +61,17 @@ function createRepl(env, cb) {
 }
 
 function setupHistory(repl, historyPath, ready) {
-  const fs = require('fs');
+  // default to using <homedir>/tmp/node_history_file
```

Comment:
Do we have any info on what people have been using for this value? This seems like something a `.node_history` file would be fine for.